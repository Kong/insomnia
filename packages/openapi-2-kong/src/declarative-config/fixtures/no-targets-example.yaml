openapi: '3.0.0'
info:
  title: Simple API overview
  version: v2
servers:
  - url: /path
paths:
  /:
    x-kong-name: x-kong-name-override-at-path-item
    get:
      x-kong-name: x-kong-name-override-at-method
      operationId: listVersionsv2
      summary: List API versions
      responses:
        '200':
          description: |-
            200 response
        '300':
          description: |-
            300 response
  /v2:
    x-kong-name: x-kong-name-override-at-path-item
    get:
      operationId: getVersionDetailsv2
      summary: Show API version details
      responses:
        '200':
          description: |-
            200 response
        '203':
          description: |-
            203 response
