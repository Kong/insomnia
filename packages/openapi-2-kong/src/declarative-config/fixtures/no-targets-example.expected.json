{
  "services": [
    {
      "protocol": "http",
      "host": "Simple_API_overview",
      "port": 80,
      "path": "/path",
      "name": "Simple_API_overview",
      "plugins": [],
      "routes": [
        {
          "methods": ["GET"],
          "strip_path": false,
          "tags": ["OAS3_import", "OAS3file_no-targets-example.yaml"],
          "name": "Simple_API_overview-x_kong_name_override_at_method",
          "paths": ["/$"]
        },
        {
          "methods": ["GET"],
          "strip_path": false,
          "tags": ["OAS3_import", "OAS3file_no-targets-example.yaml"],
          "name": "Simple_API_overview-getVersionDetailsv2",
          "paths": ["/v2$"]
        }
      ],
      "tags": ["OAS3_import", "OAS3file_no-targets-example.yaml"]
    }
  ],
  "upstreams": [
    {
      "tags": ["OAS3_import", "OAS3file_no-targets-example.yaml"],
      "name": "Simple_API_overview",
      "targets": []
    }
  ],
  "_format_version": "1.1"
}
